modules/EnsEMBL/Web/ImageConfigExtension/Tracks.pm

```diff
@@ -892,26 +892,33 @@ sub add_alignments {
 
   return unless grep $self->get_node($_), qw(multiple_align pairwise_tblat pairwise_blastz pairwise_other conservation cactus_hal_pw);
 
-  my $species_defs = $self->species_defs;
-
-  return if $species_defs->ENSEMBL_SUBTYPE eq 'Pre';
+  my $species_defs  = $self->species_defs;
+  my $alignments    = {};
+  my $self_label    = $species_defs->species_label($species, 'no_formatting');
+  my $prod_name     = $species_defs->get_config($species, 'SPECIES_PRODUCTION_NAME');
+  my $lookup        = $species_defs->prodnames_to_urls_lookup;
 
-  my $alignments = {};
-  my $self_label = $species_defs->species_label($species, 'no_formatting');
-  my $static     = '/info/genome/compara/analyses.html';
-  my $prod_name  = $species_defs->get_config($species, 'SPECIES_PRODUCTION_NAME');
-  my $map        = $species_defs->multi_val('ENSEMBL_SPECIES_URL_MAP');
-  my $comp_key   = $prod_name;
+  my ($static, $wga_page);
+  my $division = $species_defs->EG_DIVISION;
+  if ($division) {
+    $wga_page = $static = 'whole_genome_alignment.html';
```

Comment:
should it be `$static . 'whole_gen....html'`?

Reply:
No - discussed in code review.

Reply:
Yes.